recipes/survindel2/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "survindel2" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kensung-lab/SurVIndel2/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 8463d4b5f5060f26520d8a57f162f74708e0a3ecb68675dec6af671688a21a3f
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('survindel2', max_pin='x.x') }}
+  skip: True  # [osx]
```

Comment:
Any reason why you skip OSX?

Reply:
I include an external library (kdtree.h and .c) that fails to compile here. The problem are some lines such as

type* t = malloc(...).

malloc returns a void*, and the compiler in bioconda considers it an error. The code compile fine on both my linux server and my os x machine.

I can work around it using -fpermissive on linux. Any suggestions for OS X?

Reply:
On OSX we are using llvm to compile..have you tried this on your osx?

Reply:
Yes, I am using clang 11.0.